desk/main.py

```diff
@@ -33,8 +34,7 @@ def main():
             func(**funcargs)
             break  # drop out immediately, which skips the "else" below
     else:
-        assert False, "Invalid subparser! This should be impossible..."
-
+        os.system('desk --help')
```

Comment:
Codacy found an issue: [Starting a process with a shell: Seems safe, but may be changed in the future, consider rewriting without shell](https://app.codacy.com/manual/s-goldman/Dusty-Evolved-Star-Kit/pullRequest?prid=6248807)